### ISSUE TITLE
Add HRRR and RRFS analyses to regional grid-to-grid workflow

### ISSUE THREAD

**Issue by mollybsmith-noaa** [NOAA-GSL/VxIngest#452]
Jeff Beck has asked if we can verify the HRRR and RRFS against their respective and each other's analyses. We already have a workflow set up that can do this, so adding new models will be quite easy.

**Comment by ian-noaa** [comment]
Modelers need to be running RRFS_A in order for us to do this. (RRFS_A is going to be restarted when the v1 retros are complete)

RRFS_A output is 15 TB/day so ideally we'd do this with cloud-based data.

**Comment by mollybsmith-noaa** [comment]
Any and all RRFS_A verification is still blocked until the RRFS_A starts running again, but I should note that we are now verifying the various MPAS models against the HRRR analysis.

**Comment by ian-noaa** [comment]
We're waiting on confirmation to determine if this is still a priority for Jeff.

**Comment by mollybsmith-noaa** [comment]
We're not running this workflow anymore, so I'm closing this issue as no longer relevant.
